### DISCUSSION TITLE
Why my account has been flagged?

### DISCUSSION THREAD

**Discussion by uCryNet** [community/community#31353] [Other Feature Feedback, Questions, & Ideas] [answered]
> Your account has been flagged. Because of that, your profile is hidden from the public. If you believe this is a mistake, [contact support](https://github.com/contact) to have your account status reviewed.

Hello. What is it? I don't have many github account. Can you fix this? Thanks.

**Comment by uCryNet** [comment]
Hello :(

**Reply by Rijaspb** [reply]
it was some mistake. Can you fix the issue please?

**Reply by deepak-411** [reply]
Why my account has been flagged.

**Reply by deepak-411** [reply]
Please solve my issue

**Reply by aikoons** [reply]
why my account flagged

**Reply by YandexMapKit** [reply]
Please solve for me too

**Reply by MelissaWills** [reply]
Hello, please can you help me? my account has been flagged also

**Reply by maradona001** [reply]
please why is my account flagged

**Reply by SmOrtMoNke** [reply]
why is my account flagged?

**Reply by Slanka0504** [reply]
same please fix my issue

**Reply by seelawrie** [reply]
Hello, me too please

**Reply by plokacong** [reply]
hello

**Reply by ghost** [reply]
I was trying to get pics sent from son, I didn't want this app

**Reply by ParamemetricBox** [reply]
why is my account flagged?Please help me

**Reply by Rahul-sl** [reply]
Your GitHub Account can be flagged if they believe you are violating below actions.
i. Due to creating more gist’s in one day your account gets flagged.
ii. Harass, abuse, threaten, or incite violence towards any individual or group, including GitHub employees, officers, and agents, or other GitHub Users;
iii. Use of servers for any form of excessive automated bulk activity (for example, spamming), or relay any other form of unsolicited advertising or solicitation through GitHub’s servers, such as get-rich-quick schemes;

**Reply by tracher999** [reply]
my account is flagget i got it just a day wtf
and i was not spamming ore so

**Reply by diasporaTransport** [reply]
My account has been flagged too, it is a mistake as well, please restore it

**Reply by mohandassrasu** [reply]
release please

**Reply by tracher999** [reply]
i allready made a ticked a month ago and its still flagget and market as spam i dont believe enything gonna chance

**Comment by melvincwng** [comment]
@uCryNet, for account related issues, please raise a ticket and check with [GitHub Support](https://support.github.com/contact/account) directly.

FYI, based on GitHub's Terms of Service, each person can only create a maximum of 1 account. Multiple accounts per user are not allowed on GitHub.

Hope this clarifies and answers your question 😁

**Reply by freeve4h** [reply]
how do you raise a ticket?

**Reply by anka-213** [reply]
@freeve4h You click this link and fill out the form: https://support.github.com/contact/account

**Reply by StingyJack** [reply]
@melvincwng - is it different for organizations? Some employers try to require employees to have separate, company specific github logins that are not tied to anything personal.

**Comment by Museveni254** [comment]
my account has been flagged for no reason Please help

**Reply by specpoc** [reply]
I have exactly the same problem with marked... what should we do?

**Comment by eypoo** [comment]
I don’t understand what happened to my account.
Would you tell me what to do now?
Thank you very much for your support

**Comment by TNiyan** [comment]
Hello! Why was my github account flagged?
I did not do anything wrong. I will be glad if you help me.

**Comment by jurbo778** [comment]
my account has been flagged, unsure why. help please

**Comment by melek-yildirim** [comment]
I don’t understand what happened to my account (melek-yildirim). What makes an account on GitHub be "flagged"?
Would you tell me what to do now? Is it possible, or perhaps has it occurred before, for a GitHub account to be flagged due to a malicious action?
Thank you very much for your support.

**Comment by Museveni254** [comment]
Kindly contact the customer care, they will help you out. As they did to me.

**Comment by 0xUpdated** [comment]
my account flagged no reason. i just create that account and i havent another acc. im new here. help me

**Comment by JoaoLucasPSP** [comment]
I need support to remove the limitation of my account please remove the "been flagged"

**Comment by BerkUnsal** [comment]
Hello, 
I don’t understand what happened to my account.
Would you tell me what to do now?
Thank you very much for your support.
I don’t know why my account has been flagged.

**Comment by sercheir** [comment]
Hello, how resolve flagged account?

**Comment by LolitaKalagato** [comment]
My account is flagged as soon as I created the account. Please fix

**Comment by Dreamscat13** [comment]
my account has been flagged too !!! please help. Thank you.

**Comment by hoainam0707** [comment]
my account has been flagged for no reason Please help

You can't vote on a locked discussion

**Comment by Iceman13400** [comment]
how come my account has been flagged just logged back in after some time was on sick leave.

**Comment by Hweeseon** [comment]
I just made my new account and one day later it says it's flagged. What should I do?

**Comment by veronikamin** [comment]
Hello! Why was my github account flagged?
I will be glad if you help me.

**Comment by skins-app** [comment]
Same here, my account is flagged as well. I've opened a ticket with support, and no response. Can someone please help me get in.

**Comment by Trav0011** [comment]
my account seems to be flagged? I dunno why?

**Comment by tieu3tieu** [comment]
Same here, my account has been flagged. I dont know why. It was a mistake. Can you fix it?

**Comment by mrSerGL** [comment]
Hello! my account has been flagged, can you help me? Thanks

**Comment by QIran-QWorld** [comment]
My account has been flagged too
Can you fix this?
Thank  you.

**Comment by Alloycivic** [comment]
My account has been flagged, Please review

**Comment by SamaraEscudero** [comment]
My account has been flagged too, it is a mistake as well, please restore it

**Comment by farooqmuhammadhussain** [comment]
Hi,
My account has been flagged too
Can you fix this?
Thank you.

**Comment by ygor-amorim** [comment]
I'ts been almost three weeks since i opened up a ticket about my account being flagged and i got no answer whatsoever. I can't log in in third party websites such as leetcode or codewars with my GitHub account due to this. I need help and an explanation!

**Comment by YandexMapKit** [comment]
Hello! Please solve my ploblem, my account has been flagged

**Comment by maradona001** [comment]
please why is my account flagged

**Comment by herlafrance** [comment]
My account has also been flagged - unsure why.  Please review, thanks!

**Comment by quintal2** [comment]
flagging goblins have taken over github, evidently

**Comment by 0xUpdated** [comment]
hello. i didnt any post or something. i just created account. its flagged.
i didnt learn or looing around. i dont have any harrasement or cheating idea

**Comment by xavierfontanilles** [comment]
I have the same problem. I follow a training. It is very distturbing. Can you fix it ?

Xavier

**Comment by romaMirsoian1998** [comment]
My account has also been flagged - unsure why. Please review, thanks!

**Comment by Triyank-jay** [comment]
Same issue My account has also been flagged  i don't know why please check asap.

**Comment by diasporaTransport** [comment]
My account has been flagged too, it is a mistake as well, please restore it

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! As a reminder, flagged account and related queries that are account-specific should go to our lovely [support team](https://support.github.com/). They're the ones who are best-able to assist in these situations. The community team and its members are unable to assist here so I'm going to lock this post!
